scripts/store_test.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+# -------------------------------------------------------------------------
+#
+#  Part of the CodeChecker project, under the Apache License v2.0 with
+#  LLVM Exceptions. See LICENSE for license information.
+#  SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# -------------------------------------------------------------------------
+"""
+Measures store times of the specified Codechecker releases with the specified
+results directory.
+"""
+
+
+import subprocess
+import tempfile
+import re
+import time
+from timeit import default_timer as timer
+
+from pathlib import Path
+from argparse import ArgumentParser
+
+import random
+
+
+def main(args):
+    """
+    Run Codechecker stores and measure elapsed times.
+    """
+    timesheet = {}
+
+    random.seed()
+    for binary in args.binaries:
+        version_p = subprocess.run([binary, "version"], stdout=subprocess.PIPE)
+        m = re.search(
+            r"(Base package version\D*)(\d+.\d+.\d+)",
+            version_p.stdout.decode("utf8")
+        )
+        version = m.group(2) if m.group(2) else binary
+
+        timesheet[version] = []
+        for count in range(args.count):
+            with tempfile.TemporaryDirectory() as tmpdir:
+                server = subprocess.Popen([binary, "server", "-w", tmpdir])
+                time.sleep(3)  # wait for server spin up.
```

Comment:
This is brittle. Could you instead repeatedly poll the output of `CodeChecker cmd runs` or something?